R/cmapplot.R

```diff
@@ -95,7 +95,7 @@ cmapplot_globals <- list(
 
   ## Establish plotting constants in bigpts (1/72 of inch)
   consts = list(
-    lwd_originline = 1,
+    lwd_originline = 1.6,
```

Comment:
Question - this had been 1.6 before because nothing below that would render when exported at 72 dpi. However, now that we are at higher DPI, I think it does work when we do it at a smaller size - is there a reason you changed it back, @matthewstern ?

Reply:
I was weirded out by this as well. When running the example code in `default_aes.R`, the `hline` showed up as the same width as the `gridline` -- AKA invisble. I had to change it back to see a difference. This was just in the plots viewer--didn't test with any PDF exports.

Do you see something different if you change it back?

Reply:
Ah, yes. So it doesn't render properly in the plot, but it does render properly when exported to PDF and PNG. For example: 
![foo](https://user-images.githubusercontent.com/58892984/99110925-c84ea080-25b0-11eb-8162-058052561e23.png)

Reply:
I would suggest that we tend toward the value that meets comms specifications (`1`), and just accept that the plot viewer window will not show exactly what we want. That might mean that we need to have a note about that somewhere in the documentation.